dev/io.openliberty.io.smallrye.config/src/io/openliberty/microprofile/config/internal/extension/OLSmallRyeConfigExtension.java

```diff
@@ -10,15 +10,21 @@
  *******************************************************************************/
 package io.openliberty.microprofile.config.internal.extension;
 
+import java.util.Collections;
+import java.util.Set;
+
 import javax.enterprise.inject.spi.Extension;
 
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.ConfigurationPolicy;
 
-import com.ibm.ws.cdi.extension.WebSphereCDIExtension;
-
+import io.openliberty.cdi.spi.CDIExtensionMetadata;
 import io.smallrye.config.inject.ConfigExtension;
 
-@Component(service = WebSphereCDIExtension.class, configurationPolicy = ConfigurationPolicy.IGNORE, property = { "service.vendor=IBM" }, immediate = true)
-public class OLSmallRyeConfigExtension extends ConfigExtension implements Extension, WebSphereCDIExtension {
+@Component(service = CDIExtensionMetadata.class, configurationPolicy = ConfigurationPolicy.IGNORE, property = { "service.vendor=IBM" }, immediate = true)
+public class OLSmallRyeConfigExtension implements CDIExtensionMetadata {
+    @Override
+    public Set<Class<? extends Extension>> getExtensions() {
+        return Collections.singleton(ConfigExtension.class);
+    }
```

Comment:
Not entirely sure what this is doing, or why it's in this PR?

Reply:
As part of the update to 2.2.0 I tried to remove as many build warnings as I could ... in this case it was the use of a deprecated interface.